packages/components/dialog/src/dialog-content.ts

```diff
@@ -44,6 +44,13 @@ export const dialogContentProps = buildProps({
     type: String,
     default: '',
   },
+  /**
+   * @description header's aria-level attribute
+   */
+  ariaLevel: {
+    type: String,
+    default: '1',
```

Comment:
The default value is set to 2 or 3, which should be more reasonable. What do you think?

[with_heading_role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level#with_heading_role)

Reply:
Good catch! [This article ](https://medium.com/@web-accessibility-education/which-heading-level-should-dialog-modals-have-7b3df89437f0)recommends using 2, so let's go with that If you're okay

Reply:
I agree